### PR TITLE
Moved the repository from github.com/odedia to github.com/spring-petclinic

### PR DESCRIPTION
**Pull request by odedia**

### CODE REVIEW

**Comment by odedia** [comment]
This is for the config repository

**Comment by odedia** [comment]
Fixed the zipkin database name in the brave mysql tracing parameter.

File: README.md [hunk 1]
```diff
@@ -63,12 +63,12 @@ cf cups -p '{"uri": "https://wavefront.surf", "api-token": "2e41f7cf-1111-2222-3
...
+Contine with creating the services and deploying the application's microservices. A sample is available at `scripts/deployToCloudFoundry.sh`. Note that some of the services' plans may be different in your environment, so please review before executing. For example, you want want to fork [https://github.com/spring-petclinic/spring-petclinic-cloud-config.git]https://github.com/spring-petclinic/spring-petclinic-cloud-config.git if you want to make changes to the configuration.
 
 ```
 echo "Creating Required Services..."
 {
-  cf create-service -c '{ "git": { "uri": "https://github.com/odedia/spring-petclinic-microservices-config.git", "periodic": true }, "count": 3 }' p.config-server standard config &
+  cf create-service -c '{ "git": { "uri": "https://github.com/spring-petclinic/spring-petclinic-microservices-config.git", "periodic": true }, "count": 3 }' p.config-server standard config &
```
Comment (arey) [inline README.md:71]: There is still a reference to microservices here

Comment (arey) [reply README.md:72]: I'll take in charge after the merge

**Pull request merged**